app/questionnaire/placeholder_transforms.py

```diff
@@ -125,6 +125,31 @@ def parse_date(date):
                 date, PlaceholderTransforms.input_date_format_month_year_only
             ).replace(tzinfo=tzutc())
 
+    @staticmethod
+    def add(lhs, rhs):
+        if any([isinstance(lhs, str), isinstance(rhs, str)]):
+            return int(lhs) + int(rhs)
+        return lhs + rhs
+
+    @staticmethod
+    def format_ordinal(number_to_format, determiner=None):
+        number_to_format_str = str(number_to_format)
+
+        if 11 <= number_to_format % 100 <= 13:
+            suffix = "th"
+        else:
+            suffix = {1: "st", 2: "nd", 3: "rd"}.get(number_to_format % 10, "th")
+
+        if determiner == "a_or_an":
+            a_or_an = (
+                "an"
+                if number_to_format_str.startswith("8") or number_to_format in [11, 18]
```

Comment:
The `number_to_format_str ` variable could be removed here and just use:
```python
if str(number_to_format).startswith("8") or number_to_format in [11, 18]
```

Reply:
Done, thanks👍